<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>480</width>
    <height>444</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Note+</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTextEdit" name="textBody">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>15</pointsize>
       </font>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>480</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_File"/>
    <addaction name="actionOpen_File"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionPrint"/>
    <addaction name="separator"/>
    <addaction name="actionExitApp"/>
   </widget>
   <widget class="QMenu" name="menuOption">
    <property name="title">
     <string>Options</string>
    </property>
    <widget class="QMenu" name="menuFont_size">
     <property name="title">
      <string>Font size</string>
     </property>
     <addaction name="action12px"/>
     <addaction name="action18px"/>
     <addaction name="action24px"/>
    </widget>
    <addaction name="menuFont_size"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionNormal"/>
    <addaction name="actionBold"/>
    <addaction name="actionItalic"/>
    <addaction name="actionUnderline"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_All"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuScreen">
     <property name="title">
      <string>Screen</string>
     </property>
     <addaction name="actionFull"/>
     <addaction name="actionNormal_2"/>
     <addaction name="actionMinimized"/>
    </widget>
    <addaction name="menuScreen"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuOption"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="opBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionTB_NewFile"/>
   <addaction name="actionTB_OpenFile"/>
   <addaction name="actionTB_Save"/>
   <addaction name="actionTB_Print"/>
   <addaction name="separator"/>
   <addaction name="actionTB_Undo"/>
   <addaction name="actionTB_Redo"/>
   <addaction name="actionTB_Cut"/>
   <addaction name="actionTB_Copy"/>
   <addaction name="actionTB_Paste"/>
   <addaction name="actionTB_SelectAll"/>
   <addaction name="separator"/>
   <addaction name="actionTB_Normal"/>
   <addaction name="actionTB_Bold"/>
   <addaction name="actionTB_Italic"/>
   <addaction name="actionTB_Underline"/>
  </widget>
  <action name="actionNew_File">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/newfile.png</normaloff>res/icons/newfile.png</iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionOpen_File">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/open.png</normaloff>res/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open..</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/save.png</normaloff>res/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/saveas.png</normaloff>res/icons/saveas.png</iconset>
   </property>
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="action12px">
   <property name="text">
    <string>12px</string>
   </property>
  </action>
  <action name="action18px">
   <property name="text">
    <string>18px</string>
   </property>
  </action>
  <action name="action24px">
   <property name="text">
    <string>24px</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/print.png</normaloff>res/icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionExitApp">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/exit.png</normaloff>res/icons/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/cut.png</normaloff>res/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/copy.png</normaloff>res/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/paste.png</normaloff>res/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/undo.png</normaloff>res/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/redo.png</normaloff>res/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionSelect_All">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/select.png</normaloff>res/icons/select.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionTB_Undo">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/undo.png</normaloff>res/icons/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionTB_Redo">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/redo.png</normaloff>res/icons/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionTB_Cut">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/cut.png</normaloff>res/icons/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionTB_Copy">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/copy.png</normaloff>res/icons/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionTB_Paste">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/paste.png</normaloff>res/icons/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionTB_NewFile">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/newfile.png</normaloff>res/icons/newfile.png</iconset>
   </property>
   <property name="text">
    <string>New File</string>
   </property>
  </action>
  <action name="actionTB_OpenFile">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/open.png</normaloff>res/icons/open.png</iconset>
   </property>
   <property name="text">
    <string>Open File</string>
   </property>
  </action>
  <action name="actionTB_Save">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/save.png</normaloff>res/icons/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionTB_Print">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/print.png</normaloff>res/icons/print.png</iconset>
   </property>
   <property name="text">
    <string>Print File</string>
   </property>
  </action>
  <action name="actionTB_SelectAll">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/select.png</normaloff>res/icons/select.png</iconset>
   </property>
   <property name="text">
    <string>Select All</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/bold.png</normaloff>res/icons/bold.png</iconset>
   </property>
   <property name="text">
    <string>Bold</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/italic.png</normaloff>res/icons/italic.png</iconset>
   </property>
   <property name="text">
    <string>Italic</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/underline.png</normaloff>res/icons/underline.png</iconset>
   </property>
   <property name="text">
    <string>Underline</string>
   </property>
  </action>
  <action name="actionNormal">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/text.png</normaloff>res/icons/text.png</iconset>
   </property>
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionTB_Normal">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/text.png</normaloff>res/icons/text.png</iconset>
   </property>
   <property name="text">
    <string>TB_Normal</string>
   </property>
  </action>
  <action name="actionTB_Bold">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/bold.png</normaloff>res/icons/bold.png</iconset>
   </property>
   <property name="text">
    <string>TB_Bold</string>
   </property>
  </action>
  <action name="actionTB_Italic">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/italic.png</normaloff>res/icons/italic.png</iconset>
   </property>
   <property name="text">
    <string>TB_Italic</string>
   </property>
  </action>
  <action name="actionTB_Underline">
   <property name="icon">
    <iconset>
     <normaloff>res/icons/underline.png</normaloff>res/icons/underline.png</iconset>
   </property>
   <property name="text">
    <string>TB_Underline</string>
   </property>
  </action>
  <action name="actionFull">
   <property name="text">
    <string>Full</string>
   </property>
  </action>
  <action name="actionNormal_2">
   <property name="text">
    <string>Normal</string>
   </property>
  </action>
  <action name="actionMinimized">
   <property name="text">
    <string>Minimized</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
